### PR TITLE
W-12439494-FlowDetailsPage-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
+endif::[]
+
+[calloutlist]
+. The name of the flow. 
+. Details about the flow:
+.. *Created On*: The date on which the flow was created. 
+.. *Flow ID*: The ID of the flow. 
+.. *Flow Owner*: The owner of the flow. 
+.. *Last Modified*: The date on which the flow was last modified.  
+.. *Status*: The status or the flow. The status can be either active or inactive.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:26]:
```suggestion
.. *Status*: The status or flow. The status can be either active or inactive.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+. The *Run History* of the flow:
+.. *Start Date*: The date on which the flow started.  
+.. *End Date*: The date on which the flow ended. 
+.. *Status*: The status or the flow. The status can be either _SUCCESS_ or _FAILED_.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:30]:
```suggestion
.. *Status*: The status or flow. The status can be either _SUCCESS_ or _FAILED_.  
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
+.. *Flow ID*: The ID of the flow. 
+.. *Flow Owner*: The owner of the flow. 
+.. *Last Modified*: The date on which the flow was last modified.  
+.. *Status*: The status or the flow. The status can be either active or inactive.
+. The *Run History* of the flow:
+.. *Start Date*: The date on which the flow started.  
+.. *End Date*: The date on which the flow ended. 
+.. *Status*: The status or the flow. The status can be either _SUCCESS_ or _FAILED_.  
+.. *Error*: If a flow fails, the error message for the failure appears.  
+. The flow actions. You can _delete_, _clone_, or _view_ an active flow.  Note that if the flow is inactive, you can _edit_ the flow.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:32]:
```suggestion
. The flow actions. You can _delete_, _clone_, or _view_ an active flow.  If the flow is inactive, you can _edit_ the flow.
```

Comment (dhirsch1) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:32]: We don't have any guidelines for using italics. The Style Guide does not mention them so we probably shouldn't use them.

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 4]
```diff
@@ -1,10 +1,37 @@
...
+ifeval::["{product}"=="salesforce"]
+image::images/flow-details-sf.png[*Flow Details* page]
+endif::[]
+
+ifeval::["{product}"=="mulesoft"]
+image::images/flow-details-ms.png[*Flow Details* page]
+endif::[]
+
+[calloutlist]
+. The name of the flow.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:20]: This list should be an unordered list.

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 5]
```diff
@@ -1,10 +1,37 @@
...
+[calloutlist]
+. The name of the flow. 
+. Details about the flow:
+.. *Created On*: The date on which the flow was created. 
+.. *Flow ID*: The ID of the flow. 
+.. *Flow Owner*: The owner of the flow. 
+.. *Last Modified*: The date on which the flow was last modified.  
+.. *Status*: The status or the flow. The status can be either active or inactive.
+. The *Run History* of the flow:
+.. *Start Date*: The date on which the flow started.
```
Comment (dhirsch1) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:28]: This sub-list should be an unordered list.

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (dhirsch1) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:30]: Per our style, we should be using backticks for statuses. We don't mention italics in our current style guidelines:

https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-fontchoices

**Review by dhirsch1 — DISMISSED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 4 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:20]: Not in this situation - this is a callout list. The numbers in the list correspond with the numbers on image.

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc]: Fixed!

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:26]: of - fixed!

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**